portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +29,26 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Sean Tisdale!</h1>");
+
+    
+    String json = messages();
```

Comment:
Since you are only using this variable once, inline it.
`response.getWriter().println(messages())`

Reply:
solved